src/types/onyx/OriginalMessage.ts

```diff
@@ -128,7 +157,18 @@ type OriginalMessagePolicyTask = {
     originalMessage: unknown;
 };
 
+type OriginalMessageModifiedExpense = {
+    actionName: typeof CONST.REPORT.ACTIONS.TYPE.MODIFIEDEXPENSE;
+    originalMessage: unknown;
+};
+
+type OriginalMessageReimbursementQueued = {
+    actionName: typeof CONST.REPORT.ACTIONS.TYPE.REIMBURSEMENTQUEUED;
+    originalMessage: unknown;
```

Comment:
@fabioh8010 Could you comment on this typing? Why is it unknown? Do we really not know the type?

Reply:
At that time we didn't know the type (same as other types previously declared here), that's why we typed as `unknown`.